Downtime script for pyops

khappone opened the pull request:
Added downtime script for pyops. Cleaned up pyops to be nicer python.

Fixed indetation

Aegil commented on pyops.py:
```diff
@@ -7,381 +7,410 @@
+        '''
+        Create downtime for the hostlist.
+
+        Lessons learned: Cloud NOT set downtime by hostid.
```
Could Cloud

khappone commented:
Btw, this now has our monitor server name in github. I thought it would be ok? but I can remove it. It makes the tool a bit more cumbersome to use though.

martbhell commented on set_downtime.py:
```diff
@@ -0,0 +1,53 @@
+    if argv is None:
+        argv = sys.argv
+
+    parser = argparse.ArgumentParser(description='Set downtime for hosts in opeview, starting now.')
```
Tiny Typo opeview.

martbhell commented on set_downtime.py:
```diff
@@ -0,0 +1,53 @@
+
+    parser = argparse.ArgumentParser(description='Set downtime for hosts in opeview, starting now.')
+    parser.add_argument('-c', '--comment', dest="comment", default="Dowtime set by pyops", help="Comment for the downtime")
+    parser.add_argument('hosts', nargs='*', help="List of hosts to apply the downtime to.")
```
I guess this list have to be space separated. With commas I got:

<pre>
$ python set_downtime.py somenode1,somenode2 -c "mycomment" -e "2018-07-17 09:58:44"
No handlers could be found for logger "pyops"
</pre>

Not sure if that's a bug that one can fix easily.

Using spaces it worked anyway :)

Are there any other formats it accepts a list of hosts in?

khappone commented:
Yes, good comments. Made the help a bit better. Also fixed logging so it'll write any problems to stderr. In the same go, I added a parameter so you can specify the starttime too.
